app/src/main/java/com/futurice/freesound/feature/search/SearchSnackbar.java

```diff
@@ -0,0 +1,23 @@
+package com.futurice.freesound.feature.search;
+
+import android.support.design.widget.Snackbar;
+import android.view.View;
+
+public class SearchSnackbar {
+    private Snackbar snackbar;
+
+    public SearchSnackbar() {
+    }
+    public void showNewSnackbar(View view, CharSequence charSequence){
+        if(snackbar != null){
+            snackbar.dismiss();
+        }
+        snackbar = Snackbar.make(view, charSequence, Snackbar.LENGTH_INDEFINITE);
```

Comment:
Do we really need to keep the reference to snackbar? For now having `LENGTH_LONG` duration would be enough